src/main/java/de/tum/in/www1/artemis/service/connectors/localci/SharedQueueManagementService.java

```diff
@@ -66,11 +65,6 @@ public class SharedQueueManagementService {
 
     private IMap<String, ZonedDateTime> dockerImageCleanupInfo;
 
-    /**
-     * Lock to prevent multiple nodes from processing the same build job.
-     */
-    private FencedLock sharedLock;
-
```

Comment:
Just checking: Is this safe to remove? It only seems to be related to canceling build jobs, but I want to be sure

Reply:
For cancelling builds, it should be safe. Since we are removing items from the queue or a map, it shouldn't matter if we send a delete request twice.